django_irods/views.py

```diff
@@ -401,6 +402,7 @@ def download(request, path, rest_call=False, use_async=True, use_reverse_proxy=T
     return response
 
 
+@swagger_auto_schema(method='get', auto_schema=None)
```

Comment:
This annotation just removes the view from the swagger view (used for _internal urls)